src/mixins/map-mixin.js

```diff
@@ -784,7 +784,7 @@ export default function mapMixin(
           .getContainer()
           .getElementsByClassName("latLonCoordinate")[0]
         latLonContainer.classList.add("visible")
-        latLonContainer.innerHTML = `Lon: ${lon}, Lat: ${lat}`
+        latLonContainer.innerHTML = `Lon: ${lon}, </br> Lat: ${lat}`
```

Comment:
Could you remove the comma here?

Reply:
Removed the comma and used the variable $gray5 for the color.